test/second_q/drivers/gaussiand/test_driver_gaussian_log.py

```diff
@@ -150,6 +150,7 @@ def test_quartic_force_constants(self):
         ]
         self.assertListEqual(qfc, expected)
 
+    @unittest.skip("HDF5 migration")
```

Comment:
This needs to be figured out once we have a format for the `WatsonHamiltonian` in the vibrational stack in place. See also #804